arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,338 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure SQL Database capabilities",
+        "description": "Describes the Azure SQL capabilities available to a subscription for a given location.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationID}/capabilities": {
+            "get": {
+                "tags": [
+                    "Capabilities"
+                ],
+                "operationId": "Capabilities_Get",
+                "description": "Returns information about the Azure SQL capabilities available for the specified region.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "locationID",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The Id of the region for which the Azure SQL capabilities are retrieved."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/LocationCapabilities"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Get location capabilities": { "$ref": "../examples/CapabilitiesGet.json" }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "MaxSizeCapability": {
+            "properties": {
+                "limit": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "The maximum size of the database (see 'unit' for the units)."
+                },
+                "unit": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The units that the limit is expressed in.",
+                    "enum": [
+                        "Megabytes",
+                        "Gigabytes",
+                        "Terabytes",
+                        "Petabytes"
+                    ],
+                    "x-ms-enum": {
+                        "name": "MaxSizeUnits",
+                        "modelAsString": true
```

Comment:
Can modelAsString be removed? It's unlikely we will go past petabytes in the foreseeable future